extension.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "One Dark Pro",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "authors": [
     "MordFustang21"
   ],
-  "description": "A port of VSCode One Dark Pro with some tweaks",
+  "description": "Theme a port of VSCode One Dark Pro with some tweaks",
```

Comment:
```suggestion
  "description": "Theme ported from VSCode One Dark Pro with some tweaks",
```

Reply:
What's the reasoning for this change? Also what you've proposed I don't think makes sense.

If this does need to change I'd prefer something along the lines of what I suggested above.